packages/terra-clinical-result-view/src/ResultView.scss

```diff
@@ -0,0 +1,125 @@
+@import './mixins';
+
+:local {
+  .result-view {
+    color: var(--result-view-background-color, #595f62);
```

Comment:
Could the variable please be renamed?  Color is for text in a control, not the background.